pyproject.toml

```diff
@@ -17,7 +17,7 @@ python = ">=3.9,<3.12"
 opensearch-py = "^2.2.0"
 pandas = "2.1.1"
 pdf2image = "^1.16.3"
-ray = { extras = ["default"], version = "^2.7.0" }
+ray = { extras = ["default"], version = "^2.7.1" }
```

Comment:
Yes. from the commit message:
pyproject.toml: Require ray 2.7.1 to avoid https://discuss.ray.io/t/ray-start-throws-valueerror-acceleratortype-should-match-v-generation-cores-chips-got/12310 Add docker dependencies, these two packages get downloaded by the importer automatically if they aren't pre-installed.